### PR TITLE
[X86][MC] Support enc/dec for IMULZU.

### PR DESCRIPTION
**Pull request by FreddyLeaf**
apx-spec: https://cdrdv2.intel.com/v1/dl/getContent/784266
apx-syntax-recommendation: https://cdrdv2.intel.com/v1/dl/getContent/817241

### CODE REVIEW

File: llvm/test/MC/X86/apx/imulzu-att.s [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# RUN: llvm-mc -triple x86_64 -show-encoding %s | FileCheck %s
```
Comment (KanRobert) [inline llvm/test/MC/X86/apx/imulzu-att.s:1]: Check error

File: llvm/lib/Target/X86/X86InstrArithmetic.td
Comment (KanRobert) [inline llvm/lib/Target/X86/X86InstrArithmetic.td:1]: Use correct indent

File: llvm/test/MC/X86/apx/imulzu-att.s (hunk 1 above)
Comment (FreddyLeaf) [reply llvm/test/MC/X86/apx/imulzu-att.s:1]: Done.

File: llvm/lib/Target/X86/X86InstrArithmetic.td
Comment (FreddyLeaf) [reply llvm/lib/Target/X86/X86InstrArithmetic.td:1]: Done.

**Review by KanRobert — APPROVED** [review]
LGTM

**Pull request merged**